components/tutorial/tutorial_tip/tutorial_tip.tsx

```diff
@@ -226,7 +248,10 @@ export default class TutorialTip extends React.PureComponent<Props, State> {
             const tag = this.props.telemetryTag + '_next';
             trackEvent('tutorial', tag);
         }
-        if (this.getLastStep(TutorialSteps[this.props.tutorialCategory || Preferences.TUTORIAL_STEP]) === this.props.currentStep) {
+
+        const category = this.props.tutorialCategory || Preferences.TUTORIAL_STEP;
+
+        if (this.getLastStep(TutorialSteps[category], category) === this.props.currentStep) {
```

Comment:
This works for non-admins right now because the last step happens to be an admin step, but if we add a non admin step after the admin START_TRIAL step, it will be broken for non-admins because they'll be pushed into the admin step, not see the tip for it, and not be able to progress to see the last step.

I'd suggest that we include logic that checks if the user is an admin or not, and if the upcoming step is only for admins and the current user is not an admin, we push them to whatever the next non-admin step is.

Reply:
@neallred that's true. I documented that this is just a temporary fix on that check in the `getLastStep`

Reply:
Maybe it's time we do that rework of the tutorial tips

Reply:
The use case growth spike is using the tutorial tip, too, so regardless of who implements the change, it might cause too much re-coding to refactor and migrate this now *and* finish the growth spike on time.

Reply:
@neallred I added the check